Gems/WhiteBox/Code/Source/Viewport/WhiteBoxPolygonTranslationModifier.cpp

```diff
@@ -20,6 +20,11 @@
 
 namespace WhiteBox
 {
+
+    AZ_CVAR(
+        float, cl_whiteBoxPolygonViewOverlapOffset, 0.004f, nullptr, AZ::ConsoleFunctorFlags::Null,
```

Comment:
what is the naming convention I don't think this is right?

Reply:
Ah sorry yes we should now use `ed_` for editor specific settings (in a separate PR feel free to update all the White Box CVars to use `ed_` instead of `cl_`, we hadn't settled on that convention when these were first added unfortunately